lib/colorado_lottery.rb

```diff
@@ -0,0 +1,45 @@
+class ColoradoLottery
+  attr_accessor :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.game_interests.include?(game.name) && (contestant.age >= 18)
+  end
+
+  def can_register?(contestant, game)
+
+    interested_and_18?(contestant, game) &&
+    ((game.national_drawing?) || (contestant.state_of_residence == "CO"))
+  end
+
+  def register_contestant(contestant, game)
+
+     if registered_contestants[game.name] == nil #&& (contestant.spending_money > game.cost)
+      registered_contestants[game.name] = []
+      registered_contestants[game.name] << contestant
+      #contestant.charge_contestant(game.cost)
+    elsif #registered_contestants[game.name] != nil #&& (contestant.spending_money > game.cost)
```

Comment:
this should be an else, not an elsif, if you're not providing an explicit condition. 

you could also do this a little bit more neatly by only shoveling once: 

```
def register_contestant(contestant, game)
    if registered_contestants[game.name] == nil
      registered_contestants[game.name] = []
    end
    registered_contestants[game.name] << contestant
end
```

Reply:
this charge_contestants logic you have commented out here was meant to be in its own method-- i see what you were thinking and it makes sense, but the interaction pattern just has this as its own thing, so we would have wanted you to stick with that